indexer/database/contract_events.go

```diff
@@ -23,19 +23,28 @@ type ContractEvent struct {
 	EventSignature common.Hash `gorm:"serializer:json"`
 	LogIndex       uint64
 	Timestamp      uint64
+
+	GethLog *types.Log `gorm:"serializer:rlp;column:rlp_bytes"`
 }
 
 func ContractEventFromGethLog(log *types.Log, timestamp uint64) ContractEvent {
+	eventSig := common.Hash{}
+	if len(log.Topics) > 0 {
+		eventSig = log.Topics[0]
```

Comment:
There is technically a difference between `common.Hash{}` and empty, there exists `LOG0` which people call the anonymous log which doesn't have any topic at all. You wouldn't be empty to tell the diff here between `LOG0` and `LOG1 null-hash`

Don't think its a huge deal

Reply:
ohh interesting, wasn't aware of this. I wanted to avoid having to deal with pointers hence the 0x0 sentinel value.

Will document this edge case

Reply:
also realize anon events with indexed fields will still have topics > 0. Just going to note that anon-events wont be well-supported in the indexer

Reply:
Wait what's the difference?